CHANGELOG.md

```diff
@@ -63,6 +63,12 @@ The file upload molecule (variation multiple) has the new class multiple added o
 `<section class="file-upload multiple">`.
 This change is needed to hide the 'No file chosen.' text on this variation.
 
+### Form-row molecule
+The form-row molecule is, instead of just two options, an array of `items`.
```

Comment:
Dit is een change in de stijlgids opbouw en enkel nuttig voor de stijlgids developers. Implementaties van de stijlgids moeten hier geen rekening mee houden, dus deze change record mag weg. Enkel echte HTML changes in de output van de stijlgids zijn belangrijk om te vernoemen.